<launch>
	<node pkg="joy" type="joy_node" name="joy_node" />
	<node pkg="seabee3_teleop" type="seabee3_teleop" name="seabee3_teleop" output="screen" >
		
		<param name="keep_alive_period" type="double" value="0.5" />
	
		<param name="speed" type="int" value="1" />
		<param name="strafe" type="int" value="0" />
		<param name="depth" type="int" value="4" />
		<param name="heading" type="int" value="3" />

		<param name="dead_man" type="int" value="4" />
		<param name="next_firing_device" type="int" value="0" />
		<param name="prev_firing_device" type="int" value="1" />
		<param name="fire_defice" type="int" value="5" />
		<param name="reset_pose" type="int" value="9" />
		
		<!-- speed in m/s -->
		<param name="speed_scale" type="double" value="1.5" />
		<param name="strafe_scale" type="double" value="0.75" />
		<param name="depth_scale" type="double" value="0.15" />
		
		<!-- speed in deg/sec -->
		<param name="heading_scale" type="double" value="90.0" />
	</node>
</launch>
